### DISCUSSION TITLE
Post Description / Subtitle

### DISCUSSION THREAD

**Discussion by w10u** [adityatelange/hugo-PaperMod#148] [Ideas]
Hi, thanks for making this theme! I thought it might be a good idea to include a description / subtitle underneath the main title, something like this.

<img width="807" alt="Screen Shot 2020-12-22 at 11 01 59 PM" src="https://user-images.githubusercontent.com/7445904/102969068-e8c02380-44a9-11eb-8462-705ffc4c0901.png">

It could be as simple as adding a `div` under title.

layout/_default/single.html
```html
<div class="post-description">
  {{ .Description }}
</div>
```

css
```css
.post-description {
    margin-top: 10px;
    margin-bottom: 5px;
}
```

I implemented this by overriding the single.html but thought it might be useful for others as well. I can create a pull request if you think it's a good idea.

**Comment by adityatelange** [comment]
@weicliu Waiting for your PR ;)

**Reply by w10u** [reply]
Sorry for the delay. Just created one: https://github.com/adityatelange/hugo-PaperMod/pull/165. Let me know if anything needs to be changed. Happy New Year!

**Reply by vovavasylyna** [reply]
hey @adityatelange, @itsweichen how do I undo this change? :) I don't want the subtitle to appear underneath the main post title.

**Reply by adityatelange** [reply]
> hey @adityatelange, @itsweichen how do I undo this change? :) I don't want the subtitle to appear underneath the main post title.

Use summary instead of description
